Looms.PoS/Looms.PoS.Application/Features/Reservation/Commands/CreateReservation/CreateReservationRequestValidator.cs

```diff
@@ -0,0 +1,38 @@
+using FluentValidation;
+using Looms.PoS.Application.Models.Requests.Reservation;
+using Looms.PoS.Application.Utilities.Validators;
+using Looms.PoS.Application.Utilities;
+
+namespace Looms.PoS.Application.Features.Reservation.Commands.CreateReservation;
+
+public class CreateReservationRequestValidator : AbstractValidator<CreateReservationRequest>
+{
+    public CreateReservationRequestValidator()
+    {
+        RuleFor(x => x.CustomerId)
+            .MustBeValidGuid();
+                    
+        RuleFor(x => x.AppointmentTime)
```

Comment:
no appointment for the same customer should exist at the same time

Reply:
validated